flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -195,9 +196,16 @@
                     .stringType()
                     .defaultValue(getDefaultFlinkImage())
                     .withDescription(
-                            "Image to use for Flink containers. "
-                                    + "The specified image must be based upon the same Apache Flink and Scala versions as used by the application. "
-                                    + "Visit https://hub.docker.com/_/flink?tab=tags for the official docker images provided by the Flink project. The Flink project also publishes docker images here: https://hub.docker.com/r/apache/flink");
+                            Description.builder()
+                                    .text(
+                                            "Image to use for Flink containers. "
+                                                    + "The specified image must be based upon the same Apache Flink and Scala versions as used by the application. "
+                                                    + "Visit %s for the official docker images provided by the Flink project. The Flink project also publishes docker images to %s.",
+                                            link("https://hub.docker.com/_/flink?tab=tags", "here"),
+                                            link(
+                                                    "https://hub.docker.com/r/apache/flink",
+                                                    "DockerHub"))
```

Comment:
```suggestion
                                                    "apache/flink DockerHub repository"))
```

Reply:
Maybe using "apache/flink DockerHub repository" is better since the official docker images are also located in the DockerHub.

Reply:
Thanks for you suggestion @wangyang0918, I have updated and submitted again, please take a look.